packages/trader/src/Modules/CFD/Components/cfd-account-card.jsx

```diff
@@ -183,36 +183,35 @@ const CFDAccountCard = ({
     const button_ref = React.useRef();
 
     React.useEffect(() => {
+        const ref_current = ref?.current;
+        const button_ref_current = button_ref?.current;
         if (existing_data) {
             const show = () => {
                 onHover?.(existing_data.group);
             };
 
-            ref.current.addEventListener('mouseenter', show);
-            button_ref?.current?.addEventListener('mouseenter', show);
+            ref_current.addEventListener('mouseenter', show);
+            button_ref_current?.addEventListener('mouseenter', show);
```

Comment:
(question) What is the reason we use `?` for button ref but not for the other ref?

Reply:
I just keep it as what it was before(wrote by @zaki) but I suppose it's because the `button_ref` is assigned to an element based on the specific condition, in the code below:
`should_show_trade_servers && (
                        <MobileWrapper>
                            <AddTradeServerButton
                                ref={button_ref}
                                onSelectAccount={onSelectAccount}
                                is_disabled={has_cfd_account_error}
                            />
                        </MobileWrapper>
                    )}`
                  so when the condition does not passed the `ref` is undefined and it results in error, since we do not  face this issue for `const ref`.

Reply:
I see. Thank you :+1: